docs/section19/19-2.md

```diff
@@ -0,0 +1,46 @@
+# Section 19 - Rule 19-2   
+**Schema Version:** 0.0.21  
+**Mandatory Rule:** True  
+**Rule ID:** 19-2     
+**Rule Description:** Baseline building plant capacities shall be based on coincident loads.    
+**Rule Assertion:** Options are Pass/Fail/IN_APPLICABLE     
+**Appendix G Section:** Section G3.1.2.2    
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each FluidLoop Data Group  
+
+**Applicability Checks:** 
+
+1. Applies to buildings with the following baseline system types: 1, 1a, 5, 5a, 7, 7a, 7b, 8, 8b, 11.1, 11b, 11.2, 11.2a, 12, 12a, 12b, 13 (buildings with chiller or boiler plants)  
+
+**Function Calls:**  
+1. get_baseline_system_types()  
+
+
+## Rule Logic:  
+**Applicability Check 1:**  
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`  
+- Check if this rule is applicable to any HVAC systems in the B_RMR, if not then the outcome is IN_APPLICABLE: `If any(hvac.id in baseline_hvac_system_dict[sys_type] for sys_type in ["SYS-1", "SYS-1a","SYS-5", "SYS-5a", "SYS-7", "SYS-7a", "SYS-7b", "SYS-8", "SYS-8b", "SYS-11.1", "SYS-11b", "SYS-11.2", "SYS-11.2a", "SYS-12", "SYS-12a", "SYS-12b", "SYS-13"]):`   
+    Create list of plant loops:
+    - For each boiler in B_RMR, save boiler to loop boiler list: `for boiler_b in B_RMR.RulesetModelInstance.boilers: loop_boiler_list.append(boiler_b.loop)`  
+    - For each chiller in B_RMR, save chiller to loop chiller list: `for chiller_b in B_RMR.RulesetModelInstance.chillers: loop_chiller_list.append(chiller_b.cooling_loop)` 
+    - Create a list of all plant loops by combining boiler and chiller lists and eliminating duplicates: `plant_loops_list_b = list(set(list(loop_chiller_list + loop_boiler_list)))`  
```

Comment:
I think we need to exclude loops that are unrelated to HVAC, e.g.  serve process heating or cooling. So I think we need to isolate loops referenced by the baseline HVAC systems and apply the logic to these loops only.

Reply:
Updated